### PR TITLE
openff-2.0.0-rc.1

### PR DESCRIPTION
**Pull request by j-wags**

### CODE REVIEW

File: openforcefields/offxml/openff-2.0.0-rc.1.offxml [hunk 1]
```diff
@@ -0,0 +1,376 @@
+<SMIRNOFF version="0.3" aromaticity_model="OEAroModel_MDL">
+	<Author>The Open Force Field Initiative</Author>
+	<Date>2021-06-07</Date>
+	<Constraints version="0.3">
+		<Constraint smirks="[#1:1]-[*:2]" id="c1"/>
+		<!-- TIP3P Water -->
+		<Constraint smirks="[#1:1]-[#8X2H2+0:2]-[#1]" id="c1" distance="0.9572 * angstrom"/>
+		<Constraint smirks="[#1:1]-[#8X2H2+0]-[#1:2]" id="c2" distance="1.5139006545247014 * angstrom"/>
```
Comment (j-wags) [inline openforcefields/offxml/openff-2.0.0-rc.1.offxml:8]: @SimonBoothroyd For the "real" release, I wonder if we want to give the TIP3P constraints distinct IDs (maybe `HB-c1` for the general hbonds and `TIP3P-c1` and `2` for the TIP3P ones)

File: openforcefields/offxml/openff-2.0.0-rc.1.offxml [hunk 2]
```diff
@@ -0,0 +1,376 @@
...
+		<Atom smirks="[#19+1:1]" epsilon="0.1936829 * mole**-1 * kilocalorie" id="n29" rmin_half="1.705 * angstrom"/>
+		<Atom smirks="[#37+1:1]" epsilon="0.3278219 * mole**-1 * kilocalorie" id="n30" rmin_half="1.813 * angstrom"/>
+		<Atom smirks="[#55+1:1]" epsilon="0.4065394 * mole**-1 * kilocalorie" id="n31" rmin_half="1.976 * angstrom"/>
+		<Atom smirks="[#9X0-1:1]" epsilon="0.003364 * mole**-1 * kilocalorie" id="n32" rmin_half="2.303 * angstrom"/>
+		<Atom smirks="[#17X0-1:1]" epsilon="0.035591 * mole**-1 * kilocalorie" id="n33" rmin_half="2.513 * angstrom"/>
+		<Atom smirks="[#35X0-1:1]" epsilon="0.0586554 * mole**-1 * kilocalorie" id="n34" rmin_half="2.608 * angstrom"/>
+		<Atom smirks="[#53X0-1:1]" epsilon="0.0536816 * mole**-1 * kilocalorie" id="n35" rmin_half="2.86 * angstrom"/>
+		<!-- TIP3P Water -->
+		<Atom smirks="[#1]-[#8X2H2+0:1]-[#1]" epsilon="0.1521 * mole**-1 * kilocalorie" id="n1" sigma="3.1507 * angstrom"/>
+		<Atom smirks="[#1:1]-[#8X2H2+0]-[#1]" epsilon="0 * mole**-1 * kilocalorie" id="n2" sigma="1 * angstrom"/>
```
Comment (j-wags) [inline openforcefields/offxml/openff-2.0.0-rc.1.offxml:358]: Same comment about the repeated IDs here -- Maybe these could become `TIP3P-n1`

**Comment by SimonBoothroyd** [comment]
@j-wags 1.3.1 also needs to go up, probably before this goes in? It may be good to push this to Monday so we can chat more in detail about it.

**Comment by j-wags** [comment]
@SimonBoothroyd I don't think any of the issues on here are critical or blocking to the sage release candidate -- Just things to note before the final release. 

But you're right about the release order. If we cut 2.0.0-rc1 before 1.3.1, then default installs will pull the 2.0.0-rc1 package and users won't get access to 1.3.1 for another release cycle.

**Comment by j-wags** [comment]
So yeah, let's touch base Monday :-)

File: openforcefields/offxml/openff-2.0.0-rc.1.offxml [hunk 3]
```diff
@@ -0,0 +1,376 @@
+<SMIRNOFF version="0.3" aromaticity_model="OEAroModel_MDL">
+    <Author>The Open Force Field Initiative</Author>
+    <Date>2021-06-07</Date>
+    <Constraints version="0.3">
+        <Constraint smirks="[#1:1]-[*:2]" id="c1"/>
+        <!-- TIP3P Water -->
+        <Constraint smirks="[#1:1]-[#8X2H2+0:2]-[#1]" id="c1" distance="0.9572 * angstrom"/>
+        <Constraint smirks="[#1:1]-[#8X2H2+0]-[#1:2]" id="c2" distance="1.5139006545247014 * angstrom"/>
+    </Constraints>
+    <Bonds version="0.3" potential="harmonic" fractional_bondorder_method="AM1-Wiberg" fractional_bondorder_interpolation="linear">
```
Comment (j-wags) [inline openforcefields/offxml/openff-2.0.0-rc.1.offxml:10]:
```suggestion
    <Bonds version="0.4" potential="harmonic" fractional_bondorder_method="AM1-Wiberg" fractional_bondorder_interpolation="linear">
```

File: openforcefields/offxml/openff-2.0.0-rc.1.offxml [hunk 4]
```diff
@@ -0,0 +1,376 @@
...
+        <Angle smirks="[*:1]=[#16X2:2]=[*:3]" angle="180.0 * degree" k="140.0 * mole**-1 * radian**-2 * kilocalorie" id="a34"/>
+        <Angle smirks="[*:1]=[#16X2:2]=[#8:3]" angle="115.1135476736 * degree" k="140.6866437418 * mole**-1 * radian**-2 * kilocalorie" id="a34a"/>
+        <Angle smirks="[#6X3:1]-[#16X2:2]-[#6X3:3]" angle="9.948379054672e+01 * degree" k="1.393977931709e+02 * mole**-1 * radian**-2 * kilocalorie" id="a35"/>
+        <Angle smirks="[#6X3:1]-[#16X2:2]-[#6X4:3]" angle="9.682091843835e+01 * degree" k="8.811084435674e+01 * mole**-1 * radian**-2 * kilocalorie" id="a36"/>
+        <Angle smirks="[#6X3:1]-[#16X2:2]-[#1:3]" angle="9.635646036688e+01 * degree" k="1.236767085425e+02 * mole**-1 * radian**-2 * kilocalorie" id="a37"/>
+        <Angle smirks="[*:1]~[#15:2]~[*:3]" angle="1.573793185985e+02 * degree" k="1.297472153764e+02 * mole**-1 * radian**-2 * kilocalorie" id="a38"/>
+    </Angles>
+    <ProperTorsions version="0.3" potential="k*(1+cos(periodicity*theta-phase))" default_idivf="auto" fractional_bondorder_method="AM1-Wiberg" fractional_bondorder_interpolation="linear">
```
Comment (j-wags) [inline openforcefields/offxml/openff-2.0.0-rc.1.offxml:142]:
```suggestion
    <ProperTorsions version="0.4" potential="k*(1+cos(periodicity*theta-phase))" default_idivf="auto" fractional_bondorder_method="AM1-Wiberg" fractional_bondorder_interpolation="linear">
```

File: openforcefields/offxml/openff_unconstrained-2.0.0-rc.1.offxml [hunk 5]
```diff
@@ -0,0 +1,370 @@
+<SMIRNOFF version="0.3" aromaticity_model="OEAroModel_MDL">
+    <Author>The Open Force Field Initiative</Author>
+    <Date>2021-06-07</Date>
+    <Bonds version="0.3" potential="harmonic" fractional_bondorder_method="AM1-Wiberg" fractional_bondorder_interpolation="linear">
```
Comment (j-wags) [inline openforcefields/offxml/openff_unconstrained-2.0.0-rc.1.offxml:4]:
```suggestion
    <Bonds version="0.4" potential="harmonic" fractional_bondorder_method="AM1-Wiberg" fractional_bondorder_interpolation="linear">
```

File: openforcefields/offxml/openff_unconstrained-2.0.0-rc.1.offxml [hunk 6]
```diff
@@ -0,0 +1,370 @@
...
+        <Angle smirks="[*:1]=[#16X2:2]=[*:3]" angle="180.0 * degree" k="140.0 * mole**-1 * radian**-2 * kilocalorie" id="a34"/>
+        <Angle smirks="[*:1]=[#16X2:2]=[#8:3]" angle="115.1135476736 * degree" k="140.6866437418 * mole**-1 * radian**-2 * kilocalorie" id="a34a"/>
+        <Angle smirks="[#6X3:1]-[#16X2:2]-[#6X3:3]" angle="9.948379054672e+01 * degree" k="1.393977931709e+02 * mole**-1 * radian**-2 * kilocalorie" id="a35"/>
+        <Angle smirks="[#6X3:1]-[#16X2:2]-[#6X4:3]" angle="9.682091843835e+01 * degree" k="8.811084435674e+01 * mole**-1 * radian**-2 * kilocalorie" id="a36"/>
+        <Angle smirks="[#6X3:1]-[#16X2:2]-[#1:3]" angle="9.635646036688e+01 * degree" k="1.236767085425e+02 * mole**-1 * radian**-2 * kilocalorie" id="a37"/>
+        <Angle smirks="[*:1]~[#15:2]~[*:3]" angle="1.573793185985e+02 * degree" k="1.297472153764e+02 * mole**-1 * radian**-2 * kilocalorie" id="a38"/>
+    </Angles>
+    <ProperTorsions version="0.3" potential="k*(1+cos(periodicity*theta-phase))" default_idivf="auto" fractional_bondorder_method="AM1-Wiberg" fractional_bondorder_interpolation="linear">
```
Comment (j-wags) [inline openforcefields/offxml/openff_unconstrained-2.0.0-rc.1.offxml:136]:
```suggestion
    <ProperTorsions version="0.4" potential="k*(1+cos(periodicity*theta-phase))" default_idivf="auto" fractional_bondorder_method="AM1-Wiberg" fractional_bondorder_interpolation="linear">
```

**Pull request merged**